base/src/main/resources/db/migration/V80__new_folder_structure.sql

```diff
@@ -0,0 +1,288 @@
+/*
+ *  Copyright (c) 2021 Vitasystems GmbH.
+ *
+ *  This file is part of project EHRbase
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *   software distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and  limitations under the License.
+ *
+ */
+
+DO
+$$
+    DECLARE
+        count_folder         INTEGER;
+        count_folder_history INTEGER;
+    BEGIN
+        ALTER TABLE ehr.folder
+            DISABLE ROW LEVEL SECURITY;
+        ALTER TABLE ehr.folder_history
+            DISABLE ROW LEVEL SECURITY;
+        SELECT count(*) FROM ehr.folder INTO count_folder;
+        SELECT count(*) FROM ehr.folder_history INTO count_folder_history;
+        ALTER TABLE ehr.folder
+            ENABLE ROW LEVEL SECURITY;
+        ALTER TABLE ehr.folder_history
+            ENABLE ROW LEVEL SECURITY;
+
+        IF count_folder != 0 or count_folder_history != 0
+        THEN
+            RAISE EXCEPTION 'Can not automatically migrate directory. See UPDATING.md';
```

Comment:
Maybe `Systems with existing ehr directories cannot be migrated automatically; See UPDATING.md`?